pick.lic

```diff
@@ -79,15 +79,26 @@ class LockPicker
         next
       end
 
-      until 'not even locked' == bput("pick my #{box} blind", 'not even locked', 'Roundtime')
-        waitrt?
-        next unless DRSkill.getxp('Locksmithing') >= 30
```

Comment:
This line appears to have checked an exp threshold. Without it, will the loop on line 83 continue even if you get above 30/34?